### PR TITLE
feat: add Matrix class to my MathLibrary

### PR DESCRIPTION
**Pull request by RogerMo01**

### CODE REVIEW

**Comment by LongJohnSilver1504** [comment]
`feat: add Matrix class to my MathLibrary` Remember use Capital case after  commit type:
feat: Add Matrix class to my MathLibrary ;)

File: MathToolsLibrary/Matrix.cs [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        public void SetElementAt(int value, int row, int column)
+        {
+            Elements[row, column] = value;
+        }
+
+        public Matrix AddMatrices(Matrix matrixA, Matrix matrixB)
+        {
+            if (matrixA.Rows != matrixB.Rows || matrixA.Columns != matrixB.Columns)
+            {
+                Console.WriteLine("Matrices cannot be added");
```
Comment (LongJohnSilver1504) [inline MathToolsLibrary/Matrix.cs]: Why not throw an exception

File: MathToolsLibrary/Matrix.cs [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                    result.Elements[i, j] = matrixA.Elements[i, j] + matrixB.Elements[i, j];
+                }
+            }
+
+            return result;
+        }
+
+        public Matrix MultiplyMatrices(Matrix matrixA, Matrix matrixB)
+        {
+            if (matrixA.Rows != matrixB.Columns)
```
Comment (LongJohnSilver1504) [inline MathToolsLibrary/Matrix.cs]: Create a generic method for handle exception... for example 
``` C#
public void HandleException(condition,message{
if(condition)
{
        throw new System.ArgumentException($"message");
}
}
```

File: MathToolsLibrary/Matrix.cs (hunk 1 above)
Comment (RogerMo01) [reply MathToolsLibrary/Matrix.cs]: Ok

Comment (RogerMo01) [reply MathToolsLibrary/Matrix.cs]: Done!

Comment (RogerMo01) [reply MathToolsLibrary/Matrix.cs]: Done!

File: MathToolsLibrary/Matrix.cs (hunk 2 above)
Comment (RogerMo01) [reply MathToolsLibrary/Matrix.cs]: Done!

**Review by LongJohnSilver1504 — APPROVED** [review]
Good Work LGTM!

**Review by LongJohnSilver1504 — APPROVED** [review]

**Comment by LongJohnSilver1504** [comment]
LGTM!

**Pull request merged**